backend/molgenis-emx2/src/main/java/org/molgenis/emx2/DatabaseListener.java

```diff
@@ -3,7 +3,7 @@
 import java.util.HashSet;
 import java.util.Set;
 
-public class DatabaseListener {
+public abstract class DatabaseListener {
```

Comment:
Would be good to add java doc describing the function this class within the system ( the why to we need the class) ( good idea for all system classes, see sun/oracle docs as example)

Reply:
I am always hesitant to add these kind of docs because they tend to get out of sync with reality. Probably naieve, but if at all possible I like the class name to explain it. Might make it more specific and call it 'DatabaseChangeListener' or something if that helps.

Reply:
I disagree but don't feel strongly, I will add a few sentences